ts/sagas/contentLoaders.ts

```diff
@@ -52,3 +58,49 @@ export function* watchContentServiceLoadSaga(): Iterator<Effect> {
     }
   });
 }
+
+/**
+ * Retrieves a municipality metadata from the static content repository
+ */
+function getMunicipalityMetadata(
```

Comment:
This can be a saga.

Reply:
@fpersico I created `watchContentMunicipalityLoadSaga` generator. See line [85](https://github.com/teamdigitale/io-app/blob/55191eaac468736a948712581dedd7e53624aa75/ts/sagas/contentLoaders.ts#L85)

Reply:
done in commit https://github.com/teamdigitale/io-app/pull/1220/commits/acd5adce7854aeeef69ed82e9e0b8570f507bdd8